scripts/taco_to_burrito.py

```diff
@@ -125,11 +130,17 @@ def parse_marker_category(marker_category_node, base=""):
 # 
 ################################################################################
 def convert_markers(xml_path: str, output_dir: str):
-    set_eprint_title(xml_path)
-    # print(xml_path)
-    tree = ET.parse(xml_path)
-    root = tree.getroot()
-
+    global error_location
+    error_location = str(xml_path)
+    try:
+        tree = ET.parse(xml_path)
+    except Exception as e:
+        eprint(ErrorType.ERROR, "Error trying to parse XML, likely incorrectly formatted:", str(e))
```

Comment:
Generally you dont want optional positional arguments to be present first in a function signature. If this is changed into a keyword argument then it should become:
```python
eprint("Error trying to parse XML, likely incorrectly formatted:", str(e), error_type=ErrorType.ERROR)
```

Reply:
locally this is what i have... guess i changed it after the commit or something